pkgs/misc/drivers/epson-tm-m30ii/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib, stdenv, fetchurl, cups, cmake }:
+
+stdenv.mkDerivation {
```

Comment:
I'd also recommend using the modern version `stdenv.mkDerivation(finalAttrs: {`.